doc/development/setup.md

```diff
@@ -0,0 +1,47 @@
+# Development Setup
+
+If you do not already have them you will need to:
+
+- [install west](https://docs.zephyrproject.org/3.6.0/develop/west/install.html)
+- [install east](https://github.com/IRNAS/irnas-east-software)
+
+Then follow these steps:
+
+```shell
+east init -m https://github.com/IRNAS/<repo-name> <repo-name>
+cd <repo-name>/project
+
+# Set up east globally (this only needs to be done once on each machine)
+east install nrfutil-toolchain-manager
+# Install toolchain for the version of NCS used in this project
+east install toolchain
+
+# Run `west update` via east to set up west modules in the repository
+east update
+
+# Create default VERSION files
+make gen-version
+```
+
+## Setup `pre-commit`
+
+Turn on `pre-commit` tool by running `pre-commit install`. If you do not have it yet **or the
+command did not succeed** follow instructions
+[here](https://github.com/IRNAS/irnas-guidelines-docs/tree/main/tools/pre-commit).
+
+## Building and flashing
+
+This repository contains several standalone firmware projects that can be build separately and are
+intended to be used for different use cases:
+
+- `app` - The main application firmware that will be used in the production. Found under `app/`
+  folder.
+- `sample` - Sample/example firmware, used either to demonstrate the use or functionality of smaller
```

Comment:
The folder used to be called `samples`. Why change?

Zephyr also has a `samples` (and `drivers`, and `tests`, ...) folder.